cmake/QueryCodePage.py

```diff
@@ -0,0 +1,25 @@
+import platform
+
+def is_windows():
+    return platform.system().lower() == "windows"
+
+if is_windows():
+    from winreg import (
+        CloseKey, OpenKey, QueryValueEx, SetValueEx,
```

Comment:
I guess the list of imports can be trimmed a bit. Doesn't look like `CloseKey`, `SetValueEx`, `HKEY_CURRENT_USER`, `KEY_ALL_ACCESS`, `REG_EXPAND_SZ`, `REG_SZ` are used.

Reply:
Yes they are not used imports. Now I correct them as suggested, force pushed.